addons/website_event/views/event_templates_list.xml

```diff
@@ -91,13 +91,13 @@
                                     <button class="accordion-button collapsed"
                                         type="button"
                                         data-bs-toggle="collapse"
-                                        t-attf-aria-controls="o_wevent_offcanvas_cat_#{category.name}"
-                                        t-att-data-bs-target="'.o_wevent_offcanvas_cat_%s' % category.name"
+                                        t-attf-aria-controls="o_wevent_offcanvas_cat_#{category.id}"
+                                        t-att-data-bs-target="'.o_wevent_offcanvas_cat_%s' % category.id"
```

Comment:
note
Could switch to attf here too while we're touching this line

Reply:
Indeed, good to know, thx